resources/views/notifications/index.blade.php

```diff
@@ -19,6 +19,12 @@
             </ul>
         </div>
         <div id="notifications-list" class="col-sm-9 col-md-7 col-md-offset-1" v-cloak>
+            @if (!$all)
+                <p v-cloak v-if="hasUnreadNotifications" class="row">
+                    <button class="btn btn-default btn-s pull-right" v-on:click="markAllAsRead({{$user_id}})">Mark all as read</button>
+                </p>
+            @endif
+            <div class="row">
```

Comment:
Why add `class="row"`? The elements overflow the `notifications-list` now.

Reply:
The button would not align with the grid borders if I didn't use it.

Reply:
For me it does exactly that with `class="row"`:
![image](https://github.com/biigle/core/assets/2457311/e883b239-2c47-4bcf-8b11-a2a529479c70)

If I remove the class and the new `<div>` it works. I have to add `class="clearfix"` to the new `<p>` though.